src/Settings/BiometricsLink/BiometricsLinkScreen.tsx

```diff
@@ -18,7 +18,8 @@ export const BiometricsLinkScreen = () => {
   const dispatch = useDispatch()
   const linkBiometricsSignIn = async () => {
     if (await canBiometricEncryptionBeEnabled()) {
-      dispatch(setSystemAuth(true))
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      ;(dispatch(setSystemAuth(true)) as any)
```

Comment:
```suggestion
      (dispatch(setSystemAuth(true)) as any)
```

Reply:
the semicolon is added automatically, it cant be removed